### ISSUE TITLE
[Feature Request] Option to hide VODcasts from directory views

### ISSUE THREAD

**Issue by iLostMyRedPill** [FrankerFaceZ/FrankerFaceZ#208]
**Web Browser**: Chrome

**Do you use BetterTTV or other Twitch extensions**: FFZ:AP

**Bug / Idea**: A checkbox in the FFZ options to remove all VODcasts from any directory view.

**Comment by dominiklohmann** [comment]
I agree with this, and please additionally hide channels that have RERUN etc. in the title (ESL channels do this a lot without enabling vodcast).
